src/components/code-snippet/on-copy.js

```diff
@@ -0,0 +1,6 @@
+export default function onCopy() {
```

Comment:
💡 This PR introduces a shared onCopy function to be used by all code snippet components. I think we could eventually do more with shared event tracks, but will save that for https://github.com/mapbox/dr-ui/issues/245